_rules/lists-follow-content-model-a73be2.md

```diff
@@ -0,0 +1,248 @@
+---
+id: a73be2
+name: List elements follow content model
+rule_type: atomic
+description: |
+  This rule checks that list elements follow content model
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility tree
+  - CSS Styling
+  - DOM Tree
+acknowledgments:
+  authors:
+		- Brian Bors
+		- Wilco Fiers
+  previous_authors:
+    - Jey Nandakumar
+---
+
+## Applicability
+
+The rule applies to every `ul`, `ol`, `dl`, and `menu` [HTML elements][] that is [included in the accessibility tree][] and has the same [semantic role][] as its [implicit semantic role][].
+
+## Expectation 1
+
+Each test target only has [child][] [node][] in the [flat tree][] for which one of the following is true:
```

Comment:
The content model for `dl` is a group of `dt` + one or more `dd`. That means we can't consider each child of the target individually and need to look at them more globally. 
Unless we want to pass the rule even if the group is not complete (fair enough), but then this differs a bit from the content model and the first bullet in the expectation is not really correct.